components/dashboard/src/Login.tsx

```diff
@@ -46,17 +48,46 @@ export function hasVisitedMarketingWebsiteBefore() {
 export function Login() {
     const { setUser } = useContext(UserContext);
     const { setTeams } = useContext(TeamsContext);
-    const showWelcome = !hasLoggedInBefore() && !hasVisitedMarketingWebsiteBefore();
+    const experiment = "login-from-context-6826";
```

Comment:
We should use the string directly whenever necessary, no need to store in a `const` here. The compiler will tell us if it's mistyped/outdated. :+1: